lib/kitchen/verifier/pester.rb

```diff
@@ -93,8 +97,38 @@ def create_sandbox
       #
       # @return [String] a command string
       def install_command
-        # prepare ps modules now so that sandbox is already created
-        really_wrap_shell_code("Write-Verbose 'Running Install Command... done.'")
+        # the sandbox has not yet been copied to the SUT.
+        install_command_string = <<-CMD
+          Write-Verbose 'Running Install Command...'
+          $modulesToRemove = @()
+          if ($#{config[:remove_inbox_packagemanagement]}) {
+            $modulesToRemove += Get-module -ListAvailable -FullyQualifiedName @{ModuleName = 'PackageManagement'; RequiredVersion = '1.0.0.1'}
+          }
+
+          if ($#{config[:remove_inbox_psget]}) {
+            $modulesToRemove += Get-module -ListAvailable -FullyQualifiedName @{ModuleName = 'PowerShellGet'; RequiredVersion = '1.0.0.1'}
+          }
+
+          if ($#{config[:remove_inbox_pester]}) {
+            $modulesToRemove += Get-module -ListAvailable -FullyQualifiedName @{ModuleName = 'Pester'; RequiredVersion = '3.4.0'}
+          }
```

Comment:
We'll do thorough review later of course but saw this and figured I'd suggest this for now anyway 💖 

```suggestion
          $modulesToRemove = @(
              if ($#{config[:remove_inbox_packagemanagement]}) {
                  Get-Module -ListAvailable -FullyQualifiedName @{ModuleName = 'PackageManagement'; RequiredVersion = '1.0.0.1'}
              }

              if ($#{config[:remove_inbox_psget]}) {
                  Get-Module -ListAvailable -FullyQualifiedName @{ModuleName = 'PowerShellGet'; RequiredVersion = '1.0.0.1'}
              }

              if ($#{config[:remove_inbox_pester]}) {
                  Get-Module -ListAvailable -FullyQualifiedName @{ModuleName = 'Pester'; RequiredVersion = '3.4.0'}
              }
          )
```

Reply:
Yeah completely agree.